ReadMe.md

```diff
@@ -1,31 +1,28 @@
-# Development Process
+PdfDroplet is a free, simple little gui tool which only does one thing:  it takes your PDF and gives you a new one, with the pages combined and reordered, ready for saving & printing as booklets.
 
-## Continuous Build System
-
-Each time code is checked in, an automatic build begins on our [TeamCity build server](http://build.palaso.org/project.html?projectId=PdfDroplet). Similarly, when there is a new version of a PDFDroplet dependency (e.g. SIL.Core), that server automatically rebuilds PDFDroplet.
+PdfDroplet.exe can also be used as a library for making booklets in other programs. [Bloom](https://github.com/BloomBooks/BloomDesktop) uses it this way.
 
-## Building C# Source Code ##
+The gui has only been release on Windows, but on Linux PdfDroplet has been used as a non-gui library.
 
-To build PDFDroplet you'll need at least a 2015 Community edition of Visual Studio.
+## Building ##
 
-Before you'll be able to build you'll have to download some binary dependencies (see below).
+In the `build` directory, run
 
-To build PDFDroplet you can open and build the solution in Visual Studio or build from the command line using msbuild.
+`buildupdate.windows.sh` or 'buildupdate.linux.sh'
 
-## Get Binary Dependencies
+(To do this on windows, you'll need some way to run bash scripts.)
 
-You will need a way to run bash scripts (e.g. Git BASH, included with [Git for Windows](https://git-for-windows.github.io/)).
+That script will fetch dependencies from our TeamCity server. It could take a minute the first time; afterwards, it will be quick as it only downloads what has changed. When you change branches, run this again.
 
-In the `build` directory, run
+Next build the solution. On Windows, use at least the 2015 Community edition of Visual Studio.
 
-`buildupdate.windows.sh`
+In order to get a preview of the output inside of PdfDroplet, you will need a PDF viewer which integrates with Internet Explorer. Acrobat Reader, PdfXchange, and FoxIt all do this.
 
-It could take a minute the first time; afterwards, it will be quick as it only downloads what has changed. When you change branches, run this again.
+## Disable Analytics
 
-#### About PdfDroplet Dependencies
+We don't want developer and tester runs (and crashes) polluting our statistics. Add the environment variable "feedback" with value "off".
```

Comment:
Should we include the instructions for disabling analytics on Linux? Or do the analytics only apply to the gui part?